### PR TITLE
grant project mover role to terra-billing

### PR DESCRIPTION
**Pull request by dvoet**
https://broadworkbench.atlassian.net/browse/CA-1314

### CODE REVIEW

**Review by andy7i — APPROVED** [review]
👍

File: scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh [hunk 1]
```diff
@@ -56,13 +56,16 @@ gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=s
 # prod only needs the terra-billing group
 if [[ "${ENV}" == "prod" ]]; then
   gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@firecloud.org --role=roles/owner
+  gcloud resource-manager folders add-iam-policy-binding "${FOLDER_ID}" --member=group:terra-billing@firecloud.org --role=roles/resourcemanager.projectMover
```
Comment (andy7i) [inline scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh:59]: just curious (maybe answered by the ticket) -- why does terra-billing need this permission? If it's to add projects to folders, could we use the CRM SA for that?

We updated all uses of CloudResourceManager to use the CloudResourceManager SA, so I just wanted to make sure we're using the right SAs for the right jobs. 
https://github.com/broadinstitute/rawls/blob/v2_workspaces/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L1276

**Review by andy7i — COMMENTED** [review]
ticket #?

File: scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh (hunk 1 above)
Comment (dvoet) [reply scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh]: looks like the CRM SA is just the Rawls SA

**Review by andy7i — APPROVED** [review]
👍

**Pull request merged**